src/components/PageSections/Header/_Header.scss

```diff
@@ -5,6 +5,12 @@
   color: $coa-color-white;
 }
 
+.coa-Header--container {
+  max-width: $coa-top-menu-max-width;
+  margin-left: auto !important;
```

Comment:
can we be more specific to avoid using the !important tag? or at least leave a note to refactor later